README.md

```diff
@@ -44,10 +44,16 @@ After checking out the repo, run `bin/setup` to install dependencies. Then, run
 
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
+To test with different versions of Rails, set the RAILS_VERSION environment variable:
```

Comment:
I'm a little nervous having a single `Gemfile` for multiple rails versions

But then you end up with multiple `Gemfile` copies and basically are back at `appraisals`.

For now this looks pretty basic.

Reply:
Yeah, the gemspec switch is very simple because we only have one gem we are switching on - once you get past that where you need multiple gems, then Appraisals is probably better.  Not worth the overhead though for this one gem.

Reply:
https://github.com/ManageIQ/activerecord-virtual_attributes/pull/106

If we want to support 7.x, lets create another branch.

Up until this week we haven't found many bugs. So just stating we are using 7.0 from now on seemed to make sense. Now with all of these changes I'm not so confident.

Maybe we can get in some big changes now, cut a branch, and then move master over to 7.0?

Or maybe it is time to go back to multiple ruby versions.

Of note, we are currently not supporting SemanticVer here. I'm surprised I can't find the discussion threads we had with NickL, Oleg, you, and maybe LJ? We did not take going away from SemVer lightly.

I've helped support quite a few active record gems. They all tend to have growing pains here and there when the code is too based upon a previous version of active record. I do believe avoiding semantic versioning for this gem makes sense.

Reply:
> If we want to support 7.x, lets create another branch.

Ok, yeah - that seems like the best approach, because if you want to test on that branch, then just switch to that branch.